### DISCUSSION TITLE
A good naming scheme -

### DISCUSSION THREAD

**Discussion by Gestas** [ankenyr/jellyfin-youtube-metadata-plugin#65] [General]
This isn't an issue, I found a way to organize videos well, thought I'd document it. I was struggling to find a way to organize the Youtube channels I'm cloning locally. A fairly new feature in `yt-dlp` makes organizing them as shows much easier. 

Starting with the 2022.01.21 release output templates support a per-video incrementing integer, `video_autonumber`. This makes it easy to build a `S01E<number>` string. I use TheFrenchGhosty's Ultimate YouTube-DL Scripts, all I needed to do was update the output - 

```
--output "$LIBRARY_PATH""/%(uploader)s/%(uploader)s - S01E%(video_autonumber)02d - %(title)s/%(uploader)s - S01E%(video_autonumber)02d - %(title)s [%(id)s].%(ext)s"
```
The important bit is `S01E%(video_autonumber)02d`. This gets a path that looks like - `<library>/mrpete222/mrpete222 - S01E06 - Restore Rusty Angle Plate #801 tubalcain last word indicator/mrpete222 - S01E06 - Restore Rusty Angle Plate #801 tubalcain last word indicator [se49jjVYMWo].mkv`. Every channel has a single season, everything is very organized and Jellyfin handles it well. 

Thanks for this project!

**Comment by ThibaultNocchi** [comment]
I'm curious, do you mind sharing how does it look in your Jellyfin? :)

**Comment by Gestas** [comment]
Of course -
![Jellyfin Home](https://user-images.githubusercontent.com/49088/182973992-eb03cf48-ff98-4b47-99a5-16524ce7eef6.png)

![IM channel](https://user-images.githubusercontent.com/49088/182973984-b55c626d-a214-463c-b0aa-28bf0267ba1c.png)

![Pico channel](https://user-images.githubusercontent.com/49088/182973999-77b17c5d-f488-4f6a-969b-3605e4301368.png)

**Comment by Gestas** [comment]
To get the episode numbering right you need to add the `--playlist-reverse` option to yt-dlp.

**Reply by crapyeah** [reply]
Do you put this in the "global custom arguments" section of ytdlp?
